### PR TITLE
docs: add cli tool for installation

### LINKED ISSUE #654: docs: Support a user-guide docs for kubectl-crane  CLI tool
## Describe the feature
We have implemented CLI tool for `crane` in gocrane/kubectl-crane#8 and gocrane/kubectl-crane#9, it supported a way to create a recommendation rule simply, adopt a recommend result  or view which related recommend results for a deployment and so on.

And we'd like to add a user-guide docs for this CLI tool, it should contain:

- How to install `kubectl-crane`? 
- How to use `kubectl-crane`? 
  We described some examples for each sub-command like https://github.com/gocrane/kubectl-crane/blob/fefd850def70a756f44d773fc932b1cf006a3f84/pkg/cmd/recommendationRule/create.go#L22-L32

cc @qmhu

### PR DESCRIPTION
**Pull request by mangoGoForward**
#### What type of PR is this?
/docs

#### What this PR does / why we need it:
add cli tool for installation

#### Which issue(s) this PR fixes:

Fixes #654

### CODE REVIEW

**Review by qmhu — COMMENTED** [review]
I think it's better to keep current navigation struct. other parts is great for me!

File: site/content/en/docs/Installation/installation-cli-tool.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+```shell
+export release=v0.2.0
+export arch=arm64
+curl -L -o kubectl-crane.tar.gz https://github.com/gocrane/kubectl-crane/releases/download/${release}/kubectl-crane_${release}_Darwin_${arch}.tar.gz
+tar -xvf kubectl-crane.tar.gz 
+cp kubectl-crane_${release}_Darwin_${arch}/kubectl-crane /usr/local/bin/
+```
+
+### Install using Krew
```
Comment (qmhu) [inline site/content/en/docs/Installation/installation-cli-tool.md:42]: we can hide this part due to the slow response.

File: site/content/zh/docs/Installation/installation-cli-tool.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+```shell
+export release=v0.2.0
+export arch=arm64
+curl -L -o kubectl-crane.tar.gz https://github.com/gocrane/kubectl-crane/releases/download/${release}/kubectl-crane_${release}_Darwin_${arch}.tar.gz
+tar -xvf kubectl-crane.tar.gz 
+cp kubectl-crane_${release}_Darwin_${arch}/kubectl-crane /usr/local/bin/
+```
+
+### 使用 krew 安装
```
Comment (qmhu) [inline site/content/zh/docs/Installation/installation-cli-tool.md:42]: the same as above.

File: site/content/en/docs/Installation/installation-cli-tool.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+`Krew` is the plugin manager for `kubectl` command-line tool.
+
+[Install and set up](https://krew.sigs.k8s.io/docs/user-guide/setup/install/) Krew on your machine.
```
Comment (qmhu) [inline site/content/en/docs/Installation/installation-cli-tool.md:46]: set up -> setup

Comment (mangoGoForward) [reply site/content/en/docs/Installation/installation-cli-tool.md]: Done

**Review by qmhu — APPROVED** [review]

**Pull request merged**